test/integration/dbt_test.py

```diff
@@ -103,7 +97,7 @@ def test_dbt_helper(pudl_io_manager: PudlMixedFormatIOManager, script_runner):
         [
             "dbt_helper",
             "add-tables",
-            "--etl-fast",
+            f"--{dbt_target}",
```

Comment:
I think it would be more intuitive if we used the same idiom with this script as dbt does, e.g. `--target etl-fast`